index.js

```diff
@@ -335,8 +335,8 @@ function onMessage(bugout, identifier, wire, message) {
         } else if (packet.y == "r") { // rpc call
           debug("rpc", identifier, packet);
           var call = packet.c.toString();
-          var argsstring = packet.a.toString();
           try {
+            var argsstring = packet["a"] ? packet.a.toString() : "null";
```

Comment:
Can you move this out of the try block?

Reply:
of course , i ll make that change next minutes..